backend/src/main/java/com/google/rolecall/services/SectionServices.java

```diff
@@ -72,31 +80,73 @@ public Section getSection(Integer id) throws EntityNotFoundException, InvalidPar
    *     or Positions have overlapping orders.
    */
   public Section createSection(SectionInfo newSection) throws InvalidParameterException {
+    Boolean isParentRevelation = newSection.type() == Section.Type.REVELATION;
     Section section = Section.newBuilder()
         .setName(newSection.name())
         .setNotes(newSection.notes())
         .setLength(newSection.length())
+        .setSiblingId(newSection.siblingId())
         .setType(newSection.type())
         .build();
 
+    Integer[] ixArray = new Integer[newSection.positions().size()];
     if(newSection.positions() != null && !newSection.positions().isEmpty()) {
       Set<Integer> orders = new HashSet<>();
+      int i = 0;
```

Comment:
Typically 'i' is short-lived iteration counter, in this case it has large scope and it's not counting every iteration but has some other meaning, let's use a semantic var name that represent what we are counting.

Reply:
It is a loop counter. I changed the name to 'loopCounter'.